app/hooks/use-money-input.ts

```diff
@@ -261,6 +265,7 @@ export function useMoneyInput({
     maxInputDecimals,
     inputValue: inputMoney,
     convertedValue: convertedMoney,
+    bitcoinInputValue: bitcoinInputMoney,
```

Comment:
I would maybe have a function called getInputValueInBtc that would do this with description that is returns input or converted value whichever one is in btc. seems more clear than this